src/Main.java

```diff
@@ -1,5 +1,104 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача №1");
+        int monthSaving = 15000;
+        float allSaving = (float) 0;
```

Comment:
```suggestion
        float allSaving = 0;
```
зачем приведение к float? число и так к нему приведётся неявно